js/templates/modals/purchase/shipping.html

```diff
@@ -10,9 +10,9 @@ <h2 class="h4 required flexExpand"><%= ob.polyT('purchase.shippingTitle') %></h2
     <% ob.userAddresses.forEach((a, i) => { %>
     <option value="<%= i %>">
       <% let part1 = `${a.name}`;
+         part1 += a.company ? `, ${a.company},` : '';
```

Comment:
Should there be a comma before the city there?

![image](https://user-images.githubusercontent.com/794517/29083336-40749d30-7c1d-11e7-88f2-acff59192c6e.png)

Some commas seem to be missing here as well.

![image](https://user-images.githubusercontent.com/794517/29083491-c678d022-7c1d-11e7-84eb-a16d4d07d0d7.png)